os/arch/arm/src/armv8-m/exc_return.h

```diff
@@ -117,11 +119,29 @@
 
 #define EXC_RETURN_STD_CONTEXT   (1 << 4)
 
-/* EXC_RETURN_HANDLER: Return to handler mode. Exception return gets state from
- * the main stack. Execution uses MSP after return.
+/* EXC_RETURN_DEF_STACKING: Default callee register stacking (DCRS).
+ * Indicates whether the default stacking rules apply, or whether the callee
+ * registers are already on the stack.  The possible values of this bit are:
+ * 0 - Stacking of the callee saved registers skipped.  1 - Default rules for
```

Comment:
How about moving the description of `1` at the next line?
It is much better to read.

Reply:
Done.